.changeset/giant-pets-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder': major
```

Comment:
```suggestion
'@backstage/plugin-scaffolder': minor
```

This should be a `minor` release as there's no breaking changes here, just a feature addition :)

Reply:
Fixed with bae1e40